ocs_ci/ocs/ui/page_objects/encryption_module.py

```
@@ -92,6 +87,64 @@ def _get_encryption_summary(self, context_key):
 
         return encryption_summary
 
+    @staticmethod
+    def _encryption_header_text(header_element):
```

Ruff (0.15.20)

[warning] Missing return type annotation for staticmethod `_encryption_header_text`

(ANN205)